cmd/tracee/main.go

```diff
@@ -0,0 +1,258 @@
+			},
+			&cli.IntFlag{
+				Name:  "blob-perf-buffer-size",
+				Value: 1024, // 4 MB of contigous pages
+				Usage: "size, in pages, of the internal perf ring buffer used to send blobs from the kernel",
+			},
+			&cli.BoolFlag{
+				Name:  "debug",
+				Value: false,
+				Usage: "write verbose debug messages to standard output and retain intermediate artifacts. enabling will output debug messages to stdout, which will likely break consumers which expect to receive machine-readable events from stdout",
+			},
+			&cli.StringFlag{
+				Name:  "install-path",
+				Value: "/tmp/tracee",
+				Usage: "path where tracee will install or lookup it's resources",
+			},
+			&cli.BoolFlag{
+				Name:  server.MetricsEndpointFlag,
+				Usage: "enable metrics endpoint",
+				Value: false,
+			},
+			&cli.BoolFlag{
+				Name:  server.HealthzEndpointFlag,
+				Usage: "enable healthz endpoint",
+				Value: false,
+			},
+			&cli.BoolFlag{
+				Name:  server.PProfEndpointFlag,
+				Usage: "enables pprof endpoints",
+				Value: false,
+			},
+			&cli.StringFlag{
+				Name:  server.ListenEndpointFlag,
+				Usage: "listening address of the metrics endpoint server",
+				Value: ":3366",
+			},
+			&cli.BoolFlag{
+				Name:  "containers",
+				Usage: "enable container info enrichment to events. this feature is experimental and may cause unexpected behavior in the pipeline",
+			},
+
+			// TODO: add webhook
+
+			// rules
+			&cli.StringFlag{
+				Name:  "rules-dir",
+				Usage: "directory where to search for rules in CEL (.yaml), OPA (.rego), and Go plugin (.so) formats",
+			},
+			&cli.StringSliceFlag{
+				Name:  "rego",
+				Usage: "Control event rego settings. run '--rego help' for more info.",
+				Value: cli.NewStringSlice(),
+			},
+			&cli.UintFlag{
+				Name:  "sig-buffer", // TODO: rename to rules-buffer?
+				Usage: "size of the event channel's buffer consumed by signatures",
+				Value: 1000,
+			},
+		},
+	}
+
+	err := app.Run(os.Args)
+	if err != nil {
+		logger.Fatal("app", "error", err)
+	}
+}
+
+func createEventsFromSignatures(sigs []detect.Signature) {
+	id := events.StartRulesID
+
+	for _, s := range sigs {
+		m, err := s.GetMetadata()
+		if err != nil {
+			logger.Error("failed to load event", "error", err)
+			continue
+		}
+
+		selectedEvents, err := s.GetSelectedEvents()
+		if err != nil {
+			logger.Error("failed to load event", "error", err)
+			continue
+		}
+
+		dependencies := make([]events.ID, 0)
+
+		for _, s := range selectedEvents {
+			eventID, found := events.Definitions.GetID(s.Name)
+			if !found {
+				logger.Error("failed to load event depedency", "event", s.Name)
+				continue
+			}
+
+			dependencies = append(dependencies, eventID)
+		}
+
+		event := events.NewEventDefinition(m.EventName, []string{"rules"}, dependencies)
+
+		events.Definitions.Add(events.ID(id), event)
+		id++
+	}
+
+}
+
+func getEventNames(c *cli.Context) []string {
+	// TODO: --trace event= will become --events
+	traceArgs := c.StringSlice("trace")
```

Comment:
As this is a new binary (and for the transition we're keeping the old), how about making it `--events` already?

Reply:
Maybe, but can be a separate PR pre-release, instead of changing this one? I feel like this PR is way too big already.

Reply:
Makes sense. 👍🏼